app/views/avo/actions/show.html.erb

```diff
@@ -51,7 +51,7 @@
             size: :sm,
             data: {
               target: :submit_action,
-            } do %>
+            }, autofocus: @fields.empty? do %>
```

Comment:
Here is where we want to ignore the hidden ones

```suggestion
            }, autofocus: @fields.reject { |field| field.is_a?(Avo::Fields::HiddenField) }.empty? do %>
```

Reply:
Good idea.